utils/useUpdatePreviewRef.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react'
+import { useRouter } from 'next/router'
+import Cookies from 'js-cookie'
+import { repoName } from 'prismic-configuration'
+
+function useGetExitPreviewRoute() {
+  const router = useRouter()
```

Comment:
There's no need to create a new instance of router, because you just did that before calling this function. Remove this and just pass in the router when you call this function.

By removing `useRouter` from this function, it will no longer be a hook and will be a normal helper function. So that's why we're removing "use" from the function name.